analysis/summary-metadata-repeated.tsv

```diff
@@ -1,31 +1,31 @@
-"JACE/Linux"	"BBBC"	"metadata"	"read"	355.5	764.078220024488	37088	93001	2368	3456	130089	2.14930582285369	365.932489451477	786.500830349406	6.50492264416315	13.9810681163127
-"JACE/Linux"	"BBBC"	"metadata"	"write"	1196	671.928880948446	37088	93001	2368	3456	130089	0.561813445609069	108.770066889632	61.1084860583931	1.93352842809365	1.08628226837038
-"JACE/Linux"	"MitoCheck"	"metadata"	"read"	332.5	207.158088628493	55363	179881	3237	122	235244	0.623031845499227	707.500751879699	440.7954991357	9.50716635338346	5.92326739861665
-"JACE/Linux"	"MitoCheck"	"metadata"	"write"	3567.5	1177.08773093419	55363	179881	3237	122	235244	0.329947506919183	65.9408549404345	21.7570206917158	0.886091888577435	0.292363809537435
-"Java/Linux"	"tubhiswt"	"metadata"	"read"	55.75	159.797857504905	2603	7805	177	215	10408	2.86632928259919	186.690582959641	535.116684722733	3.1004764573991	8.88698645985245
-"Java/Linux"	"tubhiswt"	"metadata"	"write"	121.05	258.510403944152	2603	7805	177	215	10408	2.1355671536072	85.9809995869475	183.618198552199	1.42793525402726	3.04945162597837
-"Java/Linux"	"BBBC"	"metadata"	"read"	213.3	200.655793259056	37088	93001	2368	3456	130089	0.940721018560975	609.887482419128	573.733973668911	10.8415377402719	10.1988624257959
-"Java/Linux"	"BBBC"	"metadata"	"write"	1159.4	418.597709149564	37088	93001	2368	3456	130089	0.361046842461242	112.203726065206	40.5108010082289	1.99456615490771	0.72013181230949
-"Java/Linux"	"MitoCheck"	"metadata"	"read"	299.75	169.95072815378	55363	179881	3237	122	235244	0.566974906267823	784.800667222686	444.962284737507	10.5458976230192	5.97925931632136
-"Java/Linux"	"MitoCheck"	"metadata"	"write"	3867.25	452.573910800039	55363	179881	3237	122	235244	0.11702732194713	60.829788609477	7.11874725557714	0.817411031740901	0.0956594239746784
-"Java/Windows"	"tubhiswt"	"metadata"	"read"	48.45	118.364501701979	2603	7805	177	215	10408	2.44302377093869	214.819401444788	524.808904188439	3.56762770897833	8.71579929889359
-"Java/Windows"	"tubhiswt"	"metadata"	"write"	132.7	276.214847727748	2603	7805	177	215	10408	2.08149847571777	78.4325546345139	163.257242918392	1.3025739449887	2.71130568100366
-"Java/Windows"	"BBBC"	"metadata"	"read"	227.75	178.875397678766	37088	93001	2368	3456	130089	0.785402404736625	571.192096597146	448.615646233953	10.1536772777168	7.9747225508384
-"Java/Windows"	"BBBC"	"metadata"	"write"	1381.9	407.72422310339	37088	93001	2368	3456	130089	0.295046112673414	94.13778131558	27.7749864328618	1.67342065272451	0.493736258453773
-"Java/Windows"	"MitoCheck"	"metadata"	"read"	335.9	195.946260087059	55363	179881	3237	122	235244	0.583347008297287	700.339386722239	408.540886037175	9.41093424382257	5.48984033641639
-"Java/Windows"	"MitoCheck"	"metadata"	"write"	4725.8	506.056643386676	55363	179881	3237	122	235244	0.10708380451705	49.7786618138728	5.33048849079711	0.668909562931144	0.0716293808765042
+"C++/Linux"	"5D"	"metadata"	"read"	102	42.1276256404899	NA	NA	NA	NA	NA	0.413015937651862	NA	NA	NA	NA
```

Comment:
Good question, looks like those columns have somehow been dropped, though it's certainly not intentional.  I'll double-check this.